modern/src/common/formatter.js

```diff
@@ -9,6 +9,8 @@ export const formatPosition = (value, key) => {
     case 'fixTime':
     case 'deviceTime':
     case 'serverTime':
+    case 'startTime':
```

Comment:
I'm actually thinking of breaking this down into separate formatters.

Also, start and end time is not part of position.

Reply:
Don't forget to remove this.